tools/Hooks/CheckFileSize.py

```diff
@@ -8,7 +8,7 @@
 import os
 
 # The maximum file-size for a file to be committed in kB
-max_file_size = 200.0
+max_file_size = 700.0
```

Comment:
how many files are between 200 and 700?

Reply:
Here are the sizes in bytes of the 5 largest files,
```
1726240  docs/config/cppreference-doxygen-web.tag.xml
706861  tools/Iwyu/boost-all.imp
153206  tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Krivodonova.cpp
150245  docs/config/bootstrap.min.css
129764  docs/Images/Domain/CoordinateMaps/EquatorialCompression.png
```

Which says two things,
1. There are only two files larger than 200 kB
2. I was wrong, the largest file is way bigger than 700 kB (I didn't even think to look in docs..)

To me this favors whitelisting the special-case files, because it feels wrong to take the limit all the way to ~2 MB just for that one doxygen config file... what do you think, @kidder?

Reply:
yes, this is what I was hoping would be the case; I'd like to keep the limit where it is and whitelist the exceptions

Reply:
Sounds good, I'll make this change 👍